Rock Paper Scissors/rock-paper-scissors.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rock Paper Scissors Game</title>
+</head>
+<body>
+    <script>
+        let computerScore = 0;
+        let userScore = 0;
+
+        function computerPlay() {
+            const choices = ["Rock", "Paper", "Scissors"];
+            const randomNumber = Math.floor(Math.random() * 3);
+            return choices[randomNumber];
+        }
+
+        function playRound() {
+            
+            const computerChoice = computerPlay();
+            let userInput = prompt("Type Rock, Paper or Scissors to make a choice");
+            let playerChoice = userInput.charAt(0).toUpperCase() + userInput.slice(1).toLowerCase();
+
+            console.log(`Player: ${playerChoice}`)
+            console.log(`Computer: ${computerChoice}`)
+        
+
+            if (playerChoice == computerChoice) {
+                console.log(`Its a draw! ${playerChoice} equals ${computerChoice}`);
+                
+            }
+            else {     
+                switch (playerChoice) {
+                    case "Rock":  
+                        if (computerChoice == "Scissors") {  
+                            userScore++;
```

Comment:
The same pattern that adds the score to the user/computer and console logs a message is being repeated over and over. Can we make a function and re-use this instead?

Reply:
ok refactored